### PR TITLE
[ADD] auth: Azure OAuth is not compatible

### PR DESCRIPTION
**Pull request by jev-odoo**

### CODE REVIEW

File: general/auth/azure.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+====================================
+Azure Oauth not compatible with Odoo
```
Comment (rim-odoo) [inline general/auth/azure.rst]: OAuth ;)

File: general/auth/azure.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+====================================
+
+Microsoft Azure Oauth identification is NOT compatible with Odoo at the moment.
```
Comment (rim-odoo) [inline general/auth/azure.rst]: ... due to specific requirements in Azure's OAuth implementation.

File: general/auth/azure.rst (hunk 1 above)
Comment (rim-odoo) [reply general/auth/azure.rst]: with a capital A

Comment (jev-odoo) [reply general/auth/azure.rst]: Sorry

Comment (rim-odoo) [reply general/auth/azure.rst]: no problem :D

**Comment by mart-e** [comment]
https://www.odoo.com/documentation/user/14.0/general.html
Shouldn't we call this page "Azure OAuth" instead? It's a bit confusing on the index page.
Especially the fact that we don't have a "OAuth" documentation :D

**Comment by rim-odoo** [comment]
yes, you're right, you can do it @mart-e

**Comment by jev-odoo** [comment]
@rim-odoo @mart-e 
If I remember well, it was the first entry about an OAuth documentation.
It contains only Azure at the moment, but we could(should) improve it with other OAuth topics ...
Should we then create 1 page per OAuth doc, or one OAuth page with multiple OAuth topics?

**Comment by mart-e** [comment]
It depends if somebody is working on the rest of documenting general OAuth flow (cc @AntoineVDV ?). If not, we can simply rename the page from now.
Got a bit of "Yeah a doc page for OAuth :tada:  ... hmmm or not :disappointed:  " moment

**Comment by rim-odoo** [comment]
> Nope, there is nothing planned on that topic. And I think that we can classify this attempt of creating an incentive in AL for new features as failed

Not at all.

Just an attempt to document a recurring support question

**Pull request merged**